### PR TITLE
logs: making the admin 'set log level' function a util

### PR DESCRIPTION
**Pull request by alyssawilk**
Risk Level: low (minor refactor)
Testing: existing admin tests, new E-M test
Docs Changes: n/a
Release Notes: n/a

### CODE REVIEW

**Review by abeyad — DISMISSED** [review]

File: mobile/library/java/org/chromium/net/impl/CronvoyLogger.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package org.chromium.net.impl;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import io.envoyproxy.envoymobile.engine.types.EnvoyLogger;
+
+final class CronvoyLogger implements EnvoyLogger {
```
Comment (abeyad) [inline mobile/library/java/org/chromium/net/impl/CronvoyLogger.java:8]: Question: did you intend to include this class as part of this PR?

Comment (alyssawilk) [reply mobile/library/java/org/chromium/net/impl/CronvoyLogger.java]: haha nope, just realized that and removed

**Review by abeyad — APPROVED** [review]

**Pull request merged**